src/utils/datetime.ts

```diff
@@ -25,7 +25,7 @@ export const formatDate = (
 ) => {
     const currentLocale = language === 'en' ? 'en-gb' : 'nb';
 
-    const format = short ? 'D. MMMM' : 'L';
+    const format = short ? 'D. MMMM YYYY' : 'L';
```

Comment:
Dette vil sette inn årstall alle steder hvor formatDate, med mindre jeg misforstår? Det brukes på endel forskjellige sider hvor det kanskje ikke skal være årstall (versjonstatus, innholdsliste etc).

Kunne en løsning være å innføre en attributt (boolean) for year. Så kunne denne defaulte til false for de stedene som allerede bruker formatDate?

Reply:
Ja, det vil sette inn årstall der formatDate brukes, men jeg sjekket og trodde det ble brukt kun her. Skal sjekke det du nevner.

Reply:
Sløvt av meg, jeg ser jo den blir importert flere steder.